cmd/checkpoint/main.go

```diff
@@ -311,6 +334,28 @@ func getLocalParentPods() (map[string]*v1.Pod, error) {
 	return podListToParentPods(&podList), nil
 }
 
+// getLocalRunningPods uses the /runningpods/ kubelet api to retrieve the local container runtime pod state
+func getLocalRunningPods() (map[string]*v1.Pod, error) {
+	// TODO(aaron): The kubelet api is currently secured by a self-signed cert. We should update this to actually verify at some point
```

Comment:
- Because the checkpointer is run as a static manifest - the checkpointer cannot rely on secrets or service accounts because it will run regardless of an api-server being available.

- Can't really rely on this because we can't rely on an api-server being available - this has to function only relying on local information (but makes use of the api when available).

- The CA should ultimately be available somewhere on the host (the kubelet needs it). So as long as the kubelet cert is signed by that CA we should be fine (e.g. the checkpointer already relies on the kubeconfig provided to the kubelet in `/etc/kubernetes/kubeconfig` (we'll also need to figure out our options here when we switch to the kubelet TLS bootstrap - the location change might cause issue)